user/tests.py

```diff
@@ -158,6 +158,6 @@ def test_login_success(self):
     self.login_data = {'username':'people','password':'testpassword'}
     self.login_response = self.client.post(self.login_url, self.login_data)
     self.assertEqual(self.login_response.status_code, 200)
```

Comment:
self.login_responseのステータスコードが200になってるってことはここが間違っていてリダイレクトできてないって感じですか？

Reply:
```suggestion
    self.assertEqual(self.login_response.status_code, 302)
```
正しくはこうですね！
login成功時には302responseが返されます！

https://developer.mozilla.org/ja/docs/Web/HTTP/Status/302